deps/chakrashim/src/v8arraybuffer.cc

```diff
@@ -107,20 +107,21 @@ ArrayBuffer::Contents ArrayBuffer::GetContents() {
 }
 
 // ENABLE_TTD
-void ArrayBuffer::TTDRawBufferNotifyRegisterForModification(byte* initialModPosition) {
-    JsTTDRawBufferAsyncModificationRegister(this, initialModPosition);
+void ArrayBuffer::TTDRawBufferNotifyRegisterForModification(
+  byte* initialModPosition) {
```

Comment:
In other areas I see two indents when the parameters start on the following line.  This helps keep them from blending in with the body of the function.